src/demo/renderer.js

```diff
@@ -52,19 +64,14 @@ export const render = () => {
 
   clearCanvas(state.canvas);
 
-  const timeBeforeCanSkipPredict = 5000;
-
   switch (state.currentMode) {
     case Modes.Training:
       drawFrame(state);
       drawMovingFish(state);
       break;
     case Modes.Predicting:
-      setState({
-        canSkipPredict:
-          $time() >= currentModeStartTime + timeBeforeCanSkipPredict
-      });
```

Comment:
Do we need to remove this?  In the video I found the instant switch of a button to a new one to be a bit jarring.  Unless somebody objects, I'd quite like to leave that delay in.